components/ContentSingle/ContentVideo.js

```diff
@@ -25,7 +25,6 @@ export default function ContentVideo(props = {}) {
         segmentData={props.segmentData}
         wistiaId={props?.wistiaId}
         title={props.title}
-        src={props.video.sources[0].uri}
```

Comment:
@leoalfonso14 We still want to support regular video URLs as well, so we don't want to get rid of this exactly. We'll have to find a way to support both 🤔